backend/backend/settings/common.py

```diff
@@ -160,6 +160,7 @@ def build_broker_url(user: str, password: str, host: str, port: str) -> str:
 SECURE_CONTENT_TYPE_NOSNIFF = True
 SESSION_COOKIE_HTTPONLY = True
 CSRF_COOKIE_HTTPONLY = True
+CSRF_TRUSTED_ORIGINS = json.loads(os.environ.get("CSRF_TRUSTED_ORIGINS", "[]"))
```

Comment:
Maybe you could add a comment here to detail what is expected as a value, typical things the user should put in this list ?

Reply:
Example: https://github.com/Substra/substra-backend/blob/fa637e3a74576eb45218102a3fcba5ea0af0a4f2/backend/backend/settings/deps/cors.py#L13

Reply:
Done, doc refreshed